frontend/src/metadata/cells/sliceCellUtil.ts

```diff
@@ -0,0 +1,60 @@
+import { selections } from "../../stores";
+import { get } from "svelte/store";
+
+export function selectSliceCell(e, sliceName) {
+	// Imitate selections in Vega bar charts.
+	if (
+		get(selections).slices.length === 1 &&
+		get(selections).slices.includes(sliceName)
+	) {
+		selections.update((s) => ({
+			slices: [],
+			metadata: s.metadata,
+			tags: s.tags,
+		}));
+		return;
+	}
+	if (e.shiftKey) {
```

Comment:
Can we make it so to select a slice for two models you don't have to hold shift? e.g. by default within a model just click/unclick to select/deselect instead of switching when you click on the other model

Reply:
Removed the shift key effect for these kind of "model depended slices" (slice that has two models to select).